docs/UsingPyRuntime.md

```diff
@@ -124,3 +132,137 @@ def set_entry_point(self, name: str):
         name: an entry point name.
     """
 ```
+
+# Python interface to compile models: PyOnnxMlirCompiler
+
+## Running the PyOnnxMlirCompiler interface
+
+An ONNX model can be compiled directly using the `onnx-mlir -O3 --EmitLib` command.
+The resulting library can then be executed using Python as shown in the previous
+sections. At times, it might be convenient to also compile a model directly in Python.
+This section explores the Python methods to do so.
+
+```python
+from PyOnnxMlirCompiler import OnnxMlirCompiler, OnnxMlirTarget, OnnxMlirOption
```

Comment:
I though we don't need the options anymore. Given that we nearly need to extra interface, I wonder if we could make a subclass of 
```
from PyRuntime import ExecutionSession
```
for example 
```
from PyCompileAndRuntime import CompileExecutionSession
```

where we have a new constructor that takes the .onnx input file, the options, and load and compile.

What do you think @Jiaqing-ASU?

Reply:
I agree. This is much smarter. Will do it. Thanks.

Reply:
Thanks for your ongoing work on this, much appreciated